MapleServer2/PacketHandlers/Game/ClubHandler.cs

```diff
@@ -48,13 +57,11 @@ private void HandleCreate(GameSession session, PacketReader packet)
 
             string clubName = packet.ReadUnicodeString();
 
-            long clubId = 0; // TODO generate unique club id
+            Club club = new Club(party, clubName);
```

Comment:
How this is later done is it generates a random id, what if there's a collision? I think later on when this is saved we should call the db/mem cache and get the next available id. For now this should be fine though.